src/Makefile.am

```diff
@@ -111,12 +111,17 @@ AM_CPPFLAGS = \
 	$(NULL)
 
 AM_CFLAGS = \
+	-std=gnu99 \
```

Comment:
`gnu11` also works. Maybe we want to jump to that right away? That's what I [did in umockdev](https://github.com/martinpitt/umockdev/pull/201), and I think we don't even support ancient platforms which can't do C11. Opinions?

Reply:
I can't believe that we're debating between a 1-decade-old and a 2-decade-old vintage of a 5-decade-old language :rofl: 

C11 is fine with me, but that actually seems like a step back.  The GCC manpage mentions:

> gnu17
> gnu18
> GNU dialect of ISO C17.  This is the default for C code.


...but it's not like there's a huge difference between C17 and C11 anyway...

Meanwhile, the next highest option is `gnu2x` which is still marked experimental.

Reply:
Ah, I thought the default was still c89 or so. Let's use gnu18 then!

Reply:
Moved to gnu18.